README.md

```diff
@@ -132,6 +133,30 @@ KEYMAP_raw_binding_map='{"&bootloader": "BOOT"}' keymap parse -z zmk-config/conf
 
 Drawing parameters that are specified in the `draw_config` field can also be overridden in [the keymap YAML](KEYMAP_SPEC.md#draw_config).
 
+
+## Custom Glyphs
+Custom glyphs can be defined in the `drawing_config` block of the keymap config. After a glyph is defined it can be subsititued in place of text by surrounding the name with `$$` (i.e `$$vol_up$$`). The provided svg must only specify a viewBox, no positional or dimentional properties should be included as they are calculated for you. The height of the svg is bound by the config properties `glyph_{tap,hold,shift}_size` and width will maintain aspect ratio. (see example). To allow for custom stuing glyphs are assigned css classes `glyph` and  `<glyph name>`.
```

Comment:
```suggestion
Custom glyphs can be defined in the `drawing_config` block of the keymap config. After a glyph is defined it can be subsititued in place of text by surrounding the name with `$$` (i.e `$$vol_up$$`). The provided svg must only specify a viewBox, no positional or dimentional properties should be included as they are calculated for you. The height of the svg is bound by the config properties `glyph_{tap,hold,shifted}_size` and width will maintain aspect ratio. (see example). To allow for custom stuing glyphs are assigned css classes `glyph` and  `<glyph name>`.
```